examples/protocol_specification_ex/ledger_api.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name: ledger_api
+author: fetchai
+version: 0.1.0
+description: A protocol for ledger APIs requests and responses.
+license: Apache-2.0
+aea_version: '>=0.4.0, <0.5.0'
+speech_acts:
+  # requests
+  get_balance:
+    ledger_id: pt:str
+    address: pt:str
+  send_signed_transaction:
+    ledger_id: pt:str
+    signed_tx: pt:bytes
+  # the following could be merged together
+  is_transaction_settled:
+    ledger_id: pt:str
+    tx_digest: pt:str
+  is_transaction_valid:
+    ledger_id: pt:str
+    tx_digest: pt:str
+  get_transaction_receipt:
+    ledger_id: pt:str
+    tx_digest: pt:str
+
+  # responses
+  balance: # response to 'get_balance'
+    amount: pt:int
+  tx_digest: # response to 'send_signed_transaction'
+    digest: pt:str
+  tx_receipt: # response to 'get_transaction_receipt'
+    data: pt:dict[pt:str,pt:str]
```

Comment:
see below; probably needs to be `ct:Any` and then define a custom type `Any` similar to gym protocol.

Reply:
yes, done